src/lr_apply.rs

```diff
@@ -102,7 +97,9 @@ unsafe fn lr_stripe<BD: BitDepth>(
             p.as_mut_ptr().add(p_offset).cast(),
             stride,
             left.as_ptr().cast(),
-            lpf.as_ptr().offset(lpf_offset).cast(),
+            // NOTE: The calculated pointer may point to before the beginning of
+            // `lr_line_buf`, so we must use `wrapping_offset` here.
```

Comment:
```suggestion
            // NOTE: The calculated pointer may point to before the beginning of
            // `lr_line_buf`, so we must use `.wrapping_offset` here.
            // `.wrapping_offset` is needed since `.offset` requires the pointer is in bounds,
            // which `.wrapping_offset` does not, and delays that requirement to when the pointer is dereferenced.
```